tests/plugins/test_malformed_dependencies.py

```diff
@@ -0,0 +1,168 @@
+# Copyright (C) 2022 Greenbone Networks GmbH
+#
+# SPDX-License-Identifier: GPL-3.0-or-later
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program.  If not, see <http://www.gnu.org/licenses/>.
+
+from pathlib import Path
+
+from troubadix.plugin import LinterError
+from troubadix.plugins.malformed_dependencies import CheckMalformedDependencies
+
+from . import PluginTestCase
+
+here = Path(__file__).parent
+
+
+class CheckMalformedDependenciesTestCase(PluginTestCase):
+    def test_ok(self):
+        path = here / "file.nasl"
+        content = (
+            'script_tag(name:"cvss_base", value:"4.0");\n'
+            'script_tag(name:"summary", value:"Foo Bar.");\n'
+            'script_dependencies("example.inc", "example2.inc");\n'
```

Comment:
```suggestion
            'script_dependencies("example.nasl", "example2.nasl");\n'
```

For the sake of completeness i would change all examples to `.nasl` because `.inc` can't be dependencies at all.

Reply:
Done